pkg/sql/create_table.go

```diff
@@ -2842,6 +2842,9 @@ func incTelemetryForNewColumn(def *tree.ColumnTableDef, desc *descpb.ColumnDescr
 			telemetry.Inc(sqltelemetry.SchemaNewColumnTypeQualificationCounter("unique"))
 		}
 	}
+	if desc.HasOnUpdate() {
```

Comment:
does this code also get used for ALTER TABLE statements?

Reply:
Yep, check [here](https://github.com/cockroachdb/cockroach/blob/master/pkg/sql/add_column.go#L75).